components/Vote/Collapsible.js

```diff
@@ -23,6 +28,31 @@ const styles = {
     width: 26,
     marginLeft: -6,
     display: 'inline-block'
+  }),
+  text: css({
+    marginTop: 10,
+    marginLeft: 20,
+    ...fontStyles.sansSerifRegular16,
+    [mediaQueries.mUp]: {
+      ...fontStyles.sansSerifRegular16,
+      lineHeight: 1.4
+    },
+    '& strong': {
+      fontFamily: fontFamilies.sansSerifMedium,
```

Comment:
> The old fontFamilies export is deprecated and should no longer be used.

You'd should use `fontStyles` instead which contains the correct `fontWeight`. Or the above suggested `Interaction.fontRule` that can be merged into the rule and already contains a bold, italic and bold italic child rules.